src/pages/Settings/Settings.tsx

```diff
@@ -53,6 +67,13 @@ export const Settings = ({ userId }: SettingsProps) => {
       >
         Data
       </Typography>
+      <Divider sx={{ my: 2 }} />
+      <Switch
+        checked={playSoundOnNewMessage}
+        onChange={handlePlaySoundOnNewMessageChange}
+      />{' '}
+      Play a sound on new message
```

Comment:
Right now this is grouped under the "Data" settings section:

![Screenshot](https://user-images.githubusercontent.com/366330/192157385-74c85685-c206-4681-8ac7-df714da9f5b7.png)

However, this isn't a "Data" setting. To me this is more of a "Chat" setting. What do you think about moving this to a new "Chat" header and section above "Data" for a more clear distinction between the two?

Reply:
Yes great idea :)